### PR TITLE
balena-cli 14.5.18

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/balena-cli.rb [hunk 1]
```diff
@@ -23,16 +23,16 @@ class BalenaCli < Formula
   end
 
   # Node looks for an unversioned `python` at build-time.
-  depends_on "python@3.10" => :build
-  depends_on "node@14"
+  depends_on "python@3.11" => :build
+  depends_on "node@16"
```
Comment (cho-m) [inline Formula/balena-cli.rb:27]: As mentioned in #118960, upstream doesn't support Node > 14 (and npm > 6). This was reason why we previously kept hitting issues on bump attempts until I used `node@14` back in #97811.

These issues were reported a year ago:
- https://github.com/balena-io/balena-cli/issues/2221
- https://github.com/balena-io/balena-cli/issues/2403

Comment (chenrui333) [reply Formula/balena-cli.rb:27]: Make sense. I actually saw the those node support issues when opening another one.

Comment (chenrui333) [reply Formula/balena-cli.rb:27]: Also in there nvmrc, is also specifies as node 12
https://github.com/balena-io/balena-cli/blob/master/.nvmrc

**Comment by chenrui333** [comment]
Closing it for now.

**Pull request closed without merging**